doc/usage/configuration.rst

```diff
@@ -3642,7 +3642,7 @@ and which failures and redirects it ignores.
 
 .. confval:: linkcheck_allowed_redirects
    :type: :code-py:`dict[str, str]`
-   :default: :code-py:`{}`
+   :default: :code-py:`None`
```

Comment:
What would happen if we placed `_sentinel_lar` in here, I wonder?  (presumably it would be inaccessible from the documentation scope, and would emit some kind of error)

Reply:
From the perspective of the end user, there is no default.
```suggestion
```

Reply:
:bulb: